src/dialogs/restart/dialog-restart.ts

```diff
@@ -328,13 +341,21 @@ class DialogRestart extends LitElement {
     };
 
   private async _handleAction(ev) {
+    if (this._loadingBackupInfo) {
+      return;
+    }
+    this._loadingBackupInfo = true;
     const action = ev.currentTarget.action as
       | "restart"
       | "reboot"
       | "shutdown"
       | "restart-safe-mode";
 
-    const backupProgressMessage = this._getBackupProgressMessage();
+    const backupState = await this._loadBackupState();
```

Comment:
you should put this in a `try` with a `finally` block that sets the loading to false probably?

Reply:
`_loadBackupState` does have a try catch and returns `idle`.